extensions/amp-carousel/0.1/amp-carousel.css

```diff
@@ -100,8 +103,8 @@ amp-carousel .amp-carousel-button.amp-disabled {
 
 .-amp-slide-item-show {
   display: flex !important;
-  align-items: center;
-  justify-content: center;
+  align-items: center !important;
```

Comment:
Why not have these properties in the `.-amp-slide-item {}`? They do not depend on visibility.